liquidprompt

```diff
@@ -78,6 +78,12 @@ else
     return
 fi
 
+__lp_use_bash_preexec() {
+    # If https://github.com/rcaloras/bash-preexec is present, we can (and should, because it interferes with
+    # PROMPT_COMMAND and DEBUG) use the zsh-hook like behavior it provides.
+    [[ "${__bp_imported:-}" == "defined" ]]
```

Comment:
There is no need for the `:` in this variable substitution. `:` means to substitute if the variable is empty, but in this case what you are substituting with is the empty string.
```suggestion
    [[ ${__bp_imported-} == "defined" ]]
```

Reply:
Done in ed52671